worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/regions/plotsquared/PlotSquaredFeature.java

```diff
@@ -47,7 +47,7 @@ public PlotSquaredFeature() {
                 setupSchematicHandler();
                 setupRegionManager();
             } catch (Throwable ignored) {
-                log.debug("Please update PlotSquared: http://ci.athion.net/job/PlotSquared/");
+                log.debug("Please update PlotSquared");
```

Comment:
Since there is no (public) CI for PlotSquared anymore, I just removed the link.

Reply:
But we can add for p2 v5 on spigot.

Reply:
Right, added the link to the spigot page